copter/source/docs/frame-type-configuration.rst

```diff
@@ -21,6 +24,22 @@ Next select the frame "Type" for your vehicle. The default type is **X**.
 For Tricopters, Y6, Traditional Helicopters, Bicopters, SingleCopters and CoaxCopters the frame type is ignored.
 
 Motor Order Diagrams
-====================
+--------------------
 
 See the :ref:`Connect ESCs and Motors <connect-escs-and-motors>` page for the list of supported multicopter frames
+
+Traditional Helicopters
+=======================
+
+Traditional Helicopters run a different version of firmware. See :ref:`traditional-helicopters` for setup and tuning information.
+
+Other Configurations
+====================
+
+.. toctree::
+   :maxdepth: 1
+
+   Tricopter <tricopter>
+   SingleCopter and CoaxCopter <singlecopter-and-coaxcopter>
+   HeliQuads <heliquads>
+   SkyRocket <skyrocket>
```

Comment:
The skyrocket is either a reference frame or perhaps a peer to the 3DR Solo.  Maybe they should both be moved to the reference frame section.

Reply:
I can move them...will do

Reply:
moved skyrocket to additional info...its not a DYI reference frame